core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -383,7 +383,7 @@
    */
   public static final String CARBON_TIMESTAMP_DEFAULT_FORMAT = "yyyy-MM-dd HH:mm:ss";
   /**
-   * CARBON_TIMESTAMP_DEFAULT_FORMAT
+   * CARBON_TIMESTAMP_FORMAT,ONE PROPERTY FOR SPECIFYING DATATYPE IS TIMESTAMPE
```

Comment:
How about "Property for specifying the format of TIMESTAMP data type column. 
e.g. yyyy/MM/dd HH:mm:ss, or using CARBON_TIMESTAMP_DEFAULT_FORMAT"